### PR TITLE
#290 - home page

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
Implemented new home page design along with footer and navigation.
It should close #291.

[Desktop design](https://www.figma.com/proto/sdPYb47iLBwsA2FQMIu5BR/Blumilk?node-id=84-21&scaling=min-zoom&content-scaling=fixed) (there is no mobile design yet)

### CODE REVIEW

File: lang/pl/content.php [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
         "contact" => "Kontakt",
     ],
     "home" => [
         "section_1" => [
             "title_1" => "Oprogramowanie ",
             "title_2" => "dopasowane ",
-            "title_3" => "do Twoich potrzeb",
+            "title_3" => "do Twoich potrzeb.",
             "subtitle_1" => "Jesteśmy pasjonatami tworzenia aplikacji web/mobile oraz zaawansowanych projektów B+R.",
             "subtitle_2" => "Zmieniamy koncepcje w działające rozwiązania",
```
Comment (EwelinaSkrzypacz) [inline lang/pl/content.php:20]: Here it should be dot at the end of the sentence.

File: lang/pl/content.php [hunk 2]
```diff
@@ -23,41 +24,55 @@
...
+            "tile_3" => [
+                "title" => "Cyfryzacja biznesu i procesów",
+                "subtitle" => "Pomagamy klientom przejść przez transformację cyfrową w ich firmach.",
+            ],
+            "tile_4" => [
+                "title" => "Projekty B+R",
+                "subtitle" => "Angażujemy się w szeroki zakres popularnych zagadnień badawczych.",
+            ],
+            "tile_5" => [
+                "title" => "Interface design",
```
Comment (EwelinaSkrzypacz) [inline lang/pl/content.php:46]: All titles we have in Polish, so why this is in English?

File: lang/pl/content.php [hunk 3]
```diff
@@ -23,41 +24,55 @@
...
+            ],
+            "tile_6" => [
+                "title" => "Wykorzystanie AI",
+                "subtitle" => "Proponujemy klientom optymalizację działań z wykorzystaniem AI.",
+            ],
         ],
         "section_4" => [
-            "badge" => "zobacz jak działamy",
             "title_1" => "Case study",
             "subtitle_1" => "Dowiedz się jak nasze realizacje przekształciły wyzwania klientów w ",
```
Comment (EwelinaSkrzypacz) [inline lang/pl/content.php:56]: It should be comma after word "się".

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. On design these tiles look a bit better - all titles on design are split into two lines and it looks better, because it's unified. (just my opinion)

Design:

image


Implementation:

image

2. On design this tile with notification has smaller padding on the title and notification text is centered

Design:

image

Implementation:

image

3. Also this badge looks a bit different - on design it has mora left and right padding and text is not bold.

Design:

image


Implementation:

image

4. Again I have some problem with buttons - when I hover on the they almost overlap.

https://github.com/blumilksoftware/website/assets/56546832/08799565-554c-4d2a-9c52-437a4570eb85

5. This subtitle "Opinie naszych klientów są bardzo ważne, świadczą o jakości naszych usług i inspirują nas do nieustannego doskonalenia." is bigger than on design and on design is a bit bold. The previous subtitle (about case study) has the right size but it should be bold a bit.

6. This tile look better on design - these blobs (?) are darker and thanks to it they are more visible. Also on design the tile is wider

Design:

image


Interface:

image


7. Footer - on design "Polityka prywatności" is not bold also texts look bigger on design.

Design:

image

Implementation:

image


8. We don't have mobile design and I will not nit pick a lot about it, but I noticed that on mobile there a lot of free space near titles :point_down:  

image

Also these tiles don't look awesome. Maybe icon should be centered on top and below it the title and text? 

image


Also these button should be one by one

image

File: lang/pl/content.php [hunk 4]
```diff
@@ -4,17 +4,18 @@
...
         "career" => "Kariera",
+        "activities" => "Aktywności",
         "contact" => "Kontakt",
     ],
     "home" => [
         "section_1" => [
             "title_1" => "Oprogramowanie ",
             "title_2" => "dopasowane ",
-            "title_3" => "do Twoich potrzeb",
+            "title_3" => "do Twoich potrzeb.",
```
[the remark this answers is not in the document]
Comment (AleksandraKozubal) [reply lang/pl/content.php:18]: In the next comment you told to keep this dot, right?

File: lang/pl/content.php (hunk 2 above)
Comment (AleksandraKozubal) [reply lang/pl/content.php:46]: Did the same as is designed

**Comment by AleksandraKozubal** [comment]
> 1. On design these tiles look a bit better - all titles on design are split into two lines and it looks better, because it's unified. (just my opinion)
> 
> 
> Design:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347686189-0c1befaf-b507-456c-9bd9-70eda42134ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODYxODktMGMxYmVmYWYtYjUwNy00NTZjLTliZDktNzBlZGE0MjEzNGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA0MDRiODg5ZjhiNDZmODA1ZmM2MGY3NjlmNDkwZWUxYjU1YWUzMTJlMTQzMjRkN2JmZjRhOTNlMTk3NGQyM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.t5GKym3E5jdEJBCzbwlvEMuK1IeZP4BeLHdSV8F36hs)
> 
> Implementation:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347686241-9da28779-b246-4544-8880-18f521160c3e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODYyNDEtOWRhMjg3NzktYjI0Ni00NTQ0LTg4ODAtMThmNTIxMTYwYzNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRhNjAyZTlkYWU0MjhkOGE4OGFmNDVhY2FhMDE0NTQ1OGM1ZjRhMDBjNzg0ZjEzNzVlNWZlMzI1ZTFjODY3N2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ncN7ERtLeBOE-_KgPfAgaJ9K-Ot-qtyV-1O1MxunDMQ)
> 
>2. On design this tile with notification has smaller padding on the title and notification text is centered
> 
> 
> Design:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347687272-3a21e380-5426-4a4c-ab18-63d4d428dc6c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODcyNzItM2EyMWUzODAtNTQyNi00YTRjLWFiMTgtNjNkNGQ0MjhkYzZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5OWRhYzdiY2ZkMWYyNDI2OTM4YWM5YjBmYjQzOGNmNjg1Mzk2Yzk4ODNiZmZhNDBkMzY0ZDIwNTY1ZTBmNDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xE56PHN7V8gPakvvororqMKQ0sLNStj_A-7Gc2qk-LI)
> 
> Implementation:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347687329-e5c59dca-c1be-4da9-94d1-74b2ea6e0cf0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODczMjktZTVjNTlkY2EtYzFiZS00ZGE5LTk0ZDEtNzRiMmVhNmUwY2YwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI0YjA3Y2M2ODc2NTc5ZWExOWU1YzNkYjE5NTY4N2E0ZmM0NWE4NWIwNDI1MDU3ZTZhYWIxN2I2YjAyNmVlMjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-D3hqUdD9LMp9UjogkCmTiIsfvGSPTdidBnB8WtRpxI)
> 
>3. Also this badge looks a bit different - on design it has mora left and right padding and text is not bold.
> 
> 
> Design:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347687535-813bdc81-001a-4d67-bb9e-637da8216c66.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODc1MzUtODEzYmRjODEtMDAxYS00ZDY3LWJiOWUtNjM3ZGE4MjE2YzY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMwOWYyNjMwNTc4MDUxYmE3Y2VlNzczMDBiYmEzOWNjYjY1MWVhZGYyOGZlOGI4NjRkZDkxM2U5ZGNiNWNiYjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EN9Gid-6wCnqTEWGkQ0sgK8yXmPxIrx6mM_qUrZAmAg)
> 
> Implementation:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347687631-e7983fb6-943a-427d-8c13-86ced57873d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODc2MzEtZTc5ODNmYjYtOTQzYS00MjdkLThjMTMtODZjZWQ1Nzg3M2QzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM4MDFhMTA1MDYwNzNiMzYwMjUyZjdkODE5NGVlMmM3MzllOTU5ODU1Mjg1OGE1Yjc3NTI3YTM4NzkzODU1MDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SZPiXr1fuPK-rs2ASfA0nd7WaIYnRdcVFKtq5jeOv6A)
> 
>4. Again I have some problem with buttons - when I hover on the they almost overlap.
> 
> 
> simplescreenrecorder-2024-07-11_07.14.25.mp4
> 
>5. This subtitle "Opinie naszych klientów są bardzo ważne, świadczą o jakości naszych usług i inspirują nas do nieustannego doskonalenia." is bigger than on design and on design is a bit bold. The previous subtitle (about case study) has the right size but it should be bold a bit.
> 
>6. This tile look better on design - these blobs (?) are darker and thanks to it they are more visible. Also on design the tile is wider
> 
> 
> Design:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347688884-02e7efa1-e95d-4904-9b30-f06d22c4b8d7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODg4ODQtMDJlN2VmYTEtZTk1ZC00OTA0LTliMzAtZjA2ZDIyYzRiOGQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcwNTIwNjJhODI3YjYxZjEwY2U2NWI0NGUyOWU0ZDdkNGFlNzdmYzIyNmI3YzI2YTFlYTNhMTViODI2YWFiZmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pmBRCCvp1nTZlXdSfqKmyAXCpbyLFq3WpSN1ejO3vrE)
> 
> Interface:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347688974-ee6747e0-0e97-4a87-8967-bc59e7526742.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODg5NzQtZWU2NzQ3ZTAtMGU5Ny00YTg3LTg5NjctYmM1OWU3NTI2NzQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0ZWYwZTRlN2ZhMjI5YzVmMWFiZGY0NDNhMjA4NzQ4N2FjNjBkY2FhZWZlY2IyZTM4NTBhYmQxZGRmNzlhYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.02rXK29dFaPTBPYAhxeoyx_mMpipFkk6gWnu3uAF5_A)
> 
>7. Footer - on design "Polityka prywatności" is not bold also texts look bigger on design.
> 
> 
> Design:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347689732-cf0807e1-0ada-4fe2-ac58-0e6466b3bafc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODk3MzItY2YwODA3ZTEtMGFkYS00ZmUyLWFjNTgtMGU2NDY2YjNiYWZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlMDUwYTdhZmI4YjRmMDcwNjhkZmY0NzEwZjRmZjRkZjdmYWI4MmE4NTg3YmFhNDM3ZGFlZGQwY2YzZjk4MTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FVA5ddLI2oX0pbuDRZcA0n7mbT1g0OmTuq0BUxQ_h_k)
> 
> Implementation:
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347689792-b39e12d3-18bc-4431-be7c-b15a16a0266d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2ODk3OTItYjM5ZTEyZDMtMThiYy00NDMxLWJlN2MtYjE1YTE2YTAyNjZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0YTViY2I5ZmJmMWJmOWFmYmMxOTBkZWRjNGQzODlkY2QzYWMwYTljM2EwNDAyNjAwMThiM2FlNGM3NmY3ZGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UGLAGlidOZyiGraZxz6MDFILRMRT7A7bSkIHRWEFrJA)
> 
>8. We don't have mobile design and I will not nit pick a lot about it, but I noticed that on mobile there a lot of free space near titles 👇
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347690799-0082fb14-b556-4528-8d7c-c0005cd2cba1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2OTA3OTktMDA4MmZiMTQtYjU1Ni00NTI4LThkN2MtYzAwMDVjZDJjYmExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4Zjk1ZjBjY2ViOGE2MzY3NDBiZDMyNzdiYmUzZGE3YzdjZGFjZmY1NTM4OGI4MWYyODIyYWQ3NDQ1NDUzMjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.BQHJvPaUHB8vK9VL9JRs97wpEfDGJEqhG9mgaPXYIVo)
> 
> Also these tiles don't look awesome. Maybe icon should be centered on top and below it the title and text?
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347690969-c89fff5c-1272-4b6d-ae81-122fab73a1ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2OTA5NjktYzg5ZmZmNWMtMTI3Mi00YjZkLWFlODEtMTIyZmFiNzNhMWZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0NTYzYTEzNmFkM2ZkY2EyMTk2YTY4MmQ0OTU2ZjE1YTJlN2RiODE4YzEyZTRjZTU2MjZlYWIwY2FiN2FkNWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hnkLNi166CJO4udp4Kxcq93f4RebGuIACdAyaVdkUSA)
> 
> Also these button should be one by one
> 
> [image](https://private-user-images.githubusercontent.com/56546832/347691129-2409615e-0818-4b80-9e6e-a6b3f2733723.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2Nzc4MDAsIm5iZiI6MTcyMDY3NzUwMCwicGF0aCI6Ii81NjU0NjgzMi8zNDc2OTExMjktMjQwOTYxNWUtMDgxOC00YjgwLTllNmUtYTZiM2YyNzMzNzIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzExVDA1NTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY5YmI0ZjFjODA2ODIyMzc1NDQ5YWUxZDdmYWRmMzVlZDRiYTdjY2FkN2MyNDg3NzZhMjc5ZGRkYjljOWVlYjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Fkih7ztTxtEB7SL8SHXtzn94ndZVxxDP5XBZlTA1e9k)

1,2,3,4,5, 7, 8 fixed
6. Are you sure? Used colors are exactly the same as in the design

File: lang/pl/content.php (hunk 2 above)
Comment (EwelinaSkrzypacz) [reply lang/pl/content.php:46]: I spoke with graphic designer and we can set there title "UI/UX"

**Comment by AleksandraKozubal** [comment]
> About 6. - yeah, I'm sure. Please look at screen above and there is a difference - on design these blobs are darker

Color was right, but opacity changed that to lighter- changed that

**Review by EwelinaSkrzypacz — APPROVED** [review]
For me it's okay. As I understand the mobile version will be improved in another pr, when mobile design will be ready?

File: resources/views/layout/navigation.blade.php [hunk 5]
```diff
@@ -103,6 +105,11 @@ class="fa-solid fa-xmark text-2xl text-brand"></i>
                     {{ __("content.pages.career") }}
                 </a>
             </div>
+            <div class="w-full group hover:text-website-normal py-4 text-lg sm:text-xl text-start @if(Str::contains($current, 'activities')) text-website-normal @endif">
```
Comment (kamilpiech97) [inline resources/views/layout/navigation.blade.php:108]: Maybe `@class` should be better.

File: resources/views/layout/navigation.blade.php [hunk 6]
```diff
@@ -103,6 +105,11 @@ class="fa-solid fa-xmark text-2xl text-brand"></i>
...
+                <a href="{{ route('activities') }}" class="my-4 p-2">
+                    {{ __("content.pages.activities") }}
+                </a>
+            </div>
             <div class="w-full group hover:text-website-normal py-4 text-lg sm:text-xl text-start @if(Str::contains($current, 'contact')) text-website-normal @endif">
```
Comment (kamilpiech97) [inline resources/views/layout/navigation.blade.php:113]: same here

File: resources/views/layout/navigation.blade.php [hunk 7]
```diff
@@ -25,11 +25,13 @@ class="hover:bg-website-extra_light hover:text-website-normal rounded-3xl px-2 @
                    class="hover:bg-website-extra_light hover:text-website-normal rounded-3xl px-2 @if(Str::contains($current, 'partners')) bg-website-extra_light text-website-normal @endif">{{ __("content.pages.partners") }}</a>
                 <a href="{{ route('career') }}"
                    class="hover:bg-website-extra_light hover:text-website-normal rounded-3xl px-2 @if(Str::contains($current, 'career')) bg-website-extra_light text-website-normal @endif">{{ __("content.pages.career") }}</a>
+                <a href="{{ route('activities') }}"
+                   class="hover:bg-website-extra_light hover:text-website-normal rounded-3xl px-2 @if(Str::contains($current, 'activities')) bg-website-extra_light text-website-normal @endif">{{ __("content.pages.activities") }}</a>
                 <a href="{{ route('contact') }}"
                    class="hover:bg-website-extra_light hover:text-website-normal rounded-3xl px-2 @if(Str::contains($current, 'contact')) bg-website-extra_light text-website-normal @endif">{{ __("content.pages.contact") }}</a>
```
Comment (kamilpiech97) [inline resources/views/layout/navigation.blade.php:31]: Maybe use `@class`

**Review by kamilpiech97 — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**